server/itx/server.go

```diff
@@ -88,9 +91,17 @@ func newServer(cfg config.Config, testing bool) (*Server, error) {
 			return nil, errors.Wrap(err, "failed to add api server as subscriber")
 		}
 	}
+
+	tasks := []lifecycle.StartStopper{}
+	jobs := cronjob.NewCronJobs(cfg.Cronjob, p2pAgent, cs.Blockchain())
+	for i := range jobs {
+		tasks = append(tasks, routine.NewRecurringTask(jobs[i].Run, jobs[i].Interval()))
+	}
+
 	// TODO: explorer dependency deleted here at #1085, need to revive by migrating to api
 	chains[cs.ChainID()] = cs
 	dispatcher.AddSubscriber(cs.ChainID(), cs)
+	dispatcher.AddMsgSubscriber(apiServer)
```

Comment:
it will only be 1 status monitor, so similar to above, use `AddStatusSubscriber(cs.ChainID(), apiServer)`?